### PR TITLE
Add "Quick Start" to Sphinx docs

### PR DESCRIPTION
**Pull request by crccheck**
## Why are we doing this?

This expands the Sphinx documentation to add a "Quick start" section. The new doc was then used to write https://github.com/CondeNast/rkt-tv-summit-bot/pull/1

## Did you document your work?

dis is it

## How can someone test these changes?

I didn't do a ReadTheDocs build, but you can examine the new restructured text files within GitHub.

## What possible risks or adverse effects are there?

This begins the breaking up of docs over multiple pages. Personally, I like keeping things to one page, but if you're sticking to one page, there's not really much point to using Sphinx, the README is better for that, and we outgrew that already.

## What are the follow-up tasks?

* keep writing docs

## Are there any known issues?

none

## Did the test coverage decrease?

N/A

### CODE REVIEW

File: docs/quickstart.rst [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+Hello world
+-----------
+
+Let's make a basic echo bot.
+
+In a clean subdirectory::
+
+  npm init --force
+  npm install launch-vehicle-fbm
```
Comment (tswicegood) [inline docs/quickstart.rst]: I didn't realize `--save` was implied now. That's cool.

Comment (crccheck) [reply docs/quickstart.rst]: npm 5 started doing that. I put npm 5 as a prerequisite earlier just so I could avoid putting `--save` here out of laziness.

File: src/app.js [hunk 2]
```diff
@@ -457,9 +457,7 @@ class Messenger extends EventEmitter {
   /**
    * Send a response to the default page
    *
-   * **DEPRECATED** 1.4.0 Use :meth:`Response.reply` instead
-   *
-   * .. depreacted:: 1.4.0
```
Comment (crccheck) [inline src/app.js]: haha this never worked because I can't spell

Comment (tswicegood) [reply src/app.js]: Depreacted – the new React framework :-)

**Pull request merged**